tests/net/lib/http_server/crime/prj.conf

```diff
@@ -39,4 +42,18 @@ CONFIG_MAIN_STACK_SIZE=2048
 # Network debug config
 #CONFIG_NET_LOG=y
 
-CONFIG_UART_PIPE=n
\ No newline at end of file
+CONFIG_NO_OPTIMIZATIONS=y
+CONFIG_DEBUG=y
+CONFIG_DEBUG_INFO=y
+
+# overlay-e1000.conf
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_NET_QEMU_ETHERNET=y
+CONFIG_PCIE=y
```

Comment:
These e1000 settings should not be set for tests which typically use the dummy interface.

Reply:
Also we can use loopback interface which does not need us to create network interface in the test if we do not need to tweak the low protocol data when it is being sent/received. Anyway, there are many examples for these in the tests directory.